apps/builder/app/shared/copy-paste/utils/init-copy-paste.ts

```diff
@@ -1,5 +1,3 @@
-import { z } from "zod";
```

Comment:
Does it need to be in utils? The folder can be flat.

Reply:
I was trying to find to make clear that its a utility reused across all plugins, idk, right now probably not necessary